### PR TITLE
Start resolving lint errors

### PR DESCRIPTION
**Pull request by MartinHjelmare**
* Remove not used imports.
* Sort imports according to PEP8.
* Add docstrings and rewrite docstrings to follow PEP257.
* Remove some not used variables.
* Replace logger with print, where logger was not defined.
* Move init method first in class.
* Lint setup.py.

This PR is part of https://github.com/oeway/ImJoy/issues/120.

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
I'll rebase when things settle down.

File: imjoy/imjoyPluginEngine.py [hunk 1]
```diff
@@ -177,8 +171,8 @@ def get_drives():
     if opt.token is None or opt.token == "":
         with open(os.path.join(WORKSPACE_DIR, ".token"), "r") as f:
             opt.token = f.read()
-except Exception as e:
-    pass
+except Exception:
+    logger.debug("Failed to generate token")
```
Comment (MartinHjelmare) [inline imjoy/imjoyPluginEngine.py]: I added debug log here instead of silent pass. Is this ok?

File: imjoy/imjoyPluginEngine.py [hunk 2]
```diff
@@ -221,14 +216,14 @@ def killProcess(pid):
     if os.path.exists(pid_file):
         with open(pid_file, "r") as f:
             killProcess(int(f.read()))
-except Exception as e:
-    pass
+except Exception:
+    logger.debug("Failed to kill last process")
```
Comment (MartinHjelmare) [inline imjoy/imjoyPluginEngine.py]: See above.

File: imjoy/imjoyPluginEngine.py [hunk 3]
```diff
@@ -1142,9 +1163,9 @@ def scandir(path, type=None, recursive=False):
 
 @streamer
 async def file_sender(writer, file_path=None):
-    """
-    This function will read large file chunk by chunk and send it through HTTP
-    without reading them into memory
+    """Read a large file chunk by chunk and send it through HTTP.
+
+    Do no read the chunks into memory.
```
Comment (MartinHjelmare) [inline imjoy/imjoyPluginEngine.py]:
```suggestion
    Do not read the chunks into memory.
```

File: imjoy/imjoyUtils3.py [hunk 4]
```diff
@@ -113,31 +115,36 @@
                         args = self._unwrap(d["args"], True)
                         result = method(*args)
                         if result is not None and inspect.isawaitable(result):
-                            await reresultt
-                    except Exception as e:
+                            await result
```
Comment (MartinHjelmare) [inline imjoy/imjoyUtils3.py]: Found this bug: :smile:

File: imjoy/__main__.py [hunk 5]
```diff
@@ -106,7 +108,7 @@ def main():
             if distutils.spawn.find_executable("pip") is None:
                 git_cmd += " pip"
             if git_cmd != "":
-                logger.info("pip command failed, trying to install git and pip...")
+                print("pip command failed, trying to install git and pip...")
```
Comment (MartinHjelmare) [inline imjoy/__main__.py]: I replaced logger with print, since we don't seem to have logging available here yet.

**Comment by oeway** [comment]
Great work you are doing here, thanks!

File: imjoy/imjoyPluginEngine.py (hunk 1 above)
Comment (oeway) [reply imjoy/imjoyPluginEngine.py]: sounds good. To be more precise, here we are trying to recover a previously generated token from a file.

**Pull request merged**